Console.SportsResults.Ananttuli/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.html
+appsettings.json
```

Comment:
I've purposely excluded `appsettings.json` from being commit accidentally since it might contain credentials. Please follow the README on how to setup the config, an example config file has also been provided.

Reply:
great work in clarifying here how it should be configured!